packages/cli/src/modules/mcp/tools/utils.ts

```diff
@@ -0,0 +1,145 @@
+import type { User } from '@n8n/db';
+import type { INode } from 'n8n-workflow';
+
+import {
+	hasHttpHeaderAuthDecryptedData,
+	hasJwtPemKeyDecryptedData,
+	hasJwtSecretDecryptedData,
+} from '../mcp.typeguards';
+
+import type { CredentialsService } from '@/credentials/credentials.service';
+
+export const getWebhookDetails = async (
+	user: User,
+	webhookNodes: INode[],
+	baseUrl: string,
+	isWorkflowActive: boolean,
+	crednetialsService: CredentialsService,
+): Promise<string> => {
```

Comment:
This function combines multiple concerns:
- fetching data
- formatting data

Can we separate these concerns into separate functions